components/x-audio/src/redux/index.jsx

```diff
@@ -65,6 +80,24 @@ export default function connectPlayer (Player) {
 			if (this.playingStateAndPropsNeedSync()) {
 				this.updatePlayingStateFromProps(prevProps);
 			}
+
+			if (this.expandedPlayerRef && !this.hasSwipeDownListener) {
+				this.listenForSwipeDown(this.expandedPlayerRef);
+			}
+		}
+
+		listenForSwipeDown (expandedPlayerRef) {
+			this.hammer = new Hammer.Manager(expandedPlayerRef);
+			this.hammer.add(new Hammer.Pan({
+				direction: Hammer.DIRECTION_DOWN,
+				threshold: 0
+			}) );
+			this.hammer.on("pan", (ev) => {
```

Comment:
Pedant comment but these arguments are typically wrapped with single quotes 😄

Reply:
@benbarnett Thank you!